### PR TITLE
Migrate components sample from TizenFX.

### PR DESCRIPTION
**Pull request by huayongxu**

### CODE REVIEW

File: NUITizenGallery/Examples/ButtonTest/ButtonTest8.cs [hunk 1]
```diff
@@ -0,0 +1,539 @@
...
+            HeightSpecification = LayoutParamPolicies.MatchParent;
+
+            AppBar = new AppBar()
+            {
+                Title = "Photo Slide Sample",
+            };
+						
+            root = new View()
+            {
+                Size = new Size(1920, 1080),
```
Comment (dongsug-song) [inline NUITizenGallery/Examples/ButtonTest/ButtonTest8.cs:52]: could you change this as using Layout ?
fixed size can not be tested in various targets.

**Review by AchoWang — APPROVED** [review]
Looks good to me.

**Review by zhouleonlei — APPROVED** [review]
lgtm

**Review by dongsug-song — APPROVED** [review]
approved

**Review by zg2nets — COMMENTED** [review]
lgtm

**Pull request merged**